platform/mds/mds-test-bundle/src/main/java/org/motechproject/mds/test/domain/Discr1.java

```diff
@@ -0,0 +1,17 @@
+package org.motechproject.mds.test.domain;
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Discriminated;
+@Entity
+@Discriminated
+abstract public class Discr1 {
+    private Long id;
```

Comment:
In the Jira ticket you mentioned that entities need to define the id field. Any reason for that? The id field is always added automatically anyway.